packages/neuron-ui/src/components/WalletEditor/index.tsx

```diff
@@ -39,31 +34,20 @@ export default ({
   }, [id, initialize, wallet.name])
 
   const inputs: InputProps[] = useInputs(editor)
-  const areParamsValid = useAreParamsValid(editor.name.value, editor.newPassword.value, editor.confirmNewPassword.value)
+  const areParamsValid = useAreParamsValid(editor.name.value)
   const toggleDialog = useToggleDialog(dispatch)
 
-  const handleSubmit = useCallback(() => {
-    dispatch({
-      type: MainActions.SetDialog,
-      payload: {
-        open: true,
-      },
-    })
-  }, [dispatch])
-
   const handleConfirm = useCallback(() => {
     toggleDialog(false)
     dispatch(
       actionCreators.updateWallet({
         id: wallet.id,
-        password: editor.password.value,
-        newPassword: editor.newPassword.value,
+        password: '',
+        newPassword: '',
```

Comment:
Won't this trigger the `newPassword` check in the controller?

Reply:
yes